event-store-plans.md

```diff
@@ -0,0 +1,174 @@
+# Event Store Improvements for V7
```

Comment:
Sidenote, it'd be easier to comment if the auto-formatting in Rider would be disabled as it's randomly breaking lines by the characters limit.